app/uk/gov/hmrc/economiccrimelevyregistration/services/RegistrationValidationService.scala

```diff
@@ -133,7 +138,10 @@ class RegistrationValidationService @Inject() (clock: Clock, schemaValidator: Sc
       .orElse(registration.partnershipEntityJourneyData.flatMap(_.registration.registeredBusinessPartnerId))
       .orElse(registration.soleTraderEntityJourneyData.flatMap(_.registration.registeredBusinessPartnerId))
 
-    validateOptExists(optBusinessPartnerId, "Business partner ID")
+    registration.entityType match {
+      case Some(Other) => Valid("OK")
```

Comment:
rather than returning 'OK' like this - we can probably do it in a cleaner way using optionality instead or something like that.